neoload/1.0.0/README.md

```diff
@@ -0,0 +1,129 @@
+
+* *Name:* `neoload-webhook`
+* *Scope:* `keptn-webhook-service`
+* *Key-value pairs*: Click the new key-value pair button and add these values:
+  - *Key* = `accountToken`
+  - *Value* = The NeoLoad Web account token from the previous section
+
+The completed form should look as follows:
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/neoload/0.9.0/images/secret-configuration.png)
+
+### (Optional) Validate you have Keptn Webhook Service deployed
+
+Keptn Webhook Service was introduced with **Keptn 0.10.0** and is getting installed by default on the Control Plane. If you manually installed Keptn or upgraded manually,
+please validate and ensure that your Keptn installation runs the Keptn `webhook-service`.
+
+## Configure Keptn for NeoLoad Webhook
+NeoLoad web supports [Keptn silent webhooks](https://www.dynatrace.com/support/help/how-to-use-dynatrace/cloud-automation/lifecycle-orchestration) where the
+webhook subscription just listens for a task started or task finished event and then calls the NeoLoad API to run a performance test.
+Once the NeoLoad webhook is triggered, NeoLoad does not need to send anything back to Keptn.  Instead, the Keptn sequence will just automatically continue.
+
+You can incorporate NeoLoad webhooks into any Keptn project, but the example in this section assumes you have created a Keptn project called **demo**,
+a stage called **performance**, a service called **demoservice**, and a sequence with a task called **trigger-neoload-test**.
+
+### Step 1: Configure webhook
+
+1. In a Keptn project, go to **Uniform page** > **Uniform**, select the webhook-service, and click the *Add subscription* button.
+2. For the subscription section, fill in the following:
+   * *Task*: `trigger-neoload-test`
+   * *Task suffix*: `triggered`
+3. For the webhook configuration section, fill in the following:
+   * *Request Method*: POST
+   * *URL*: The NeoLoad API endpoint with the workspace and test ID using this syntax `https://neoload-api.saas.neotys.com/v3/workspaces/<Your WorkspaceID>/tests/<Your TestID>/start?testResultName=<Your Test Result Name>&testResultDescription=<Your test result description>`
+   * For example: `https://neoload-api.saas.neotys.com/v3/workspaces/5e3abde2e860a13ca619/tests/f61787dd-da45-44f0-9e0d-a76b4a0371e/start?testResultName=Demo-Test-Result&testResultDescription=stage-{{.data.stage}}`
+   * Refer to the [NeoLoad Web API documentation](https://neoload-api.saas.neotys.com/explore/) for more details
+   * Add a custom header with the following values:
+     - *name*: accountToken
+     - *value*: {{.secret.neoload-webhook.accountToken}}
+   * The Custom Payload field can be left empty.
+
+The completed form should look as follows:
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/neoload/0.9.0/images/webhook.png)
+
+4. Click the Create subscription button to save and enable the webhook
+
+### Step 2: Adjust webhook
```

Comment:
Correction, the "toggle" looks to have been added in keptn 0.12 .  See the bottom of this page.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/24907089/155369403-12736073-a333-4a06-8537-75ad20b8adb8.png">

Reply:
This feature is not in the documentation of Keptn 0.13.x: https://keptn.sh/docs/0.13.x/integrations/webhooks/#configure-webhook-to-not-auto-respond-with-a-finished-event
I think for now i will keep the "manual" way to set the `sendFinished` flag.
Moreover, our keptn environment is still in 0.10.x so I can't make the screenshot.

Reply:
The documents page just got updated. There was a PR that I had that took some time to get through.  See the page now.  And Cloud Automation (CA) in Dynatrace just got updated today to 0.12.2 so that toggle UI feature is now in both Keptn and CA.